ASTree.cpp

```diff
@@ -2428,6 +2445,24 @@ void print_src(PycRef<ASTNode> node, PycModule* mod)
             fputs(" }", pyc_output);
         }
         break;
+    case ASTNode::NODE_CONST_MAP:
+        {
+            PycRef<ASTConstMap> const_map = node.cast<ASTConstMap>();
+            PycTuple::value_t keys = const_map->keys().cast<ASTObject>()->object().cast<PycTuple>()->values();
+            ASTConstMap::values_t values = const_map->values();
+
+            auto map = new ASTMap;
+            for (auto key : keys) {
```

Comment:
`const auto& key` to avoid unnecessary copies

Reply:
Done!